src/listeners.js

```diff
@@ -0,0 +1,19 @@
+import { initializeScoreboard, addToScoreBoard, setDisabledState } from './dom-functions.js';
+
+export default () => {
+  const refreshBtn = document.getElementById('refresh-btn');
+  refreshBtn.addEventListener('click', initializeScoreboard);
+  refreshBtn.click();
+
+  const formElem = document.querySelector('form');
+  const userInput = document.getElementById('name');
+  const scoreInput = document.getElementById('score');
+  formElem.addEventListener('submit', async (e) => {
+    e.preventDefault();
+    const submitInput = formElem.querySelector('input[type=submit]');
+    setDisabledState(submitInput, true);
+    const success = await addToScoreBoard(userInput.value, scoreInput.value);
+    if (success) initializeScoreboard();
+    setDisabledState(submitInput, false);
```

Comment:
- [OPTIONAL] It can be good to reset the form fields after submitting the data with success.

Reply:
Thank. You. I'll work on this